src/Constants.sol

```diff
@@ -210,7 +210,7 @@ abstract contract Helper is TestBase {
         vm.label(0x32467a5fc2d72D21E8DCe990906547A2b012f382, "Constants.WBTC_FRAX_PAIR_ADDRESS");
         vm.label(0x794F6B13FBd7EB7ef10d1ED205c9a416910207Ff, "Constants.WETH_FRAX_PAIR_ADDRESS");
         vm.label(0xc9268c7F59850B26567b0f5d3Dcc317cEDC43490, "Constants.SLIPPAGE_AUCTION_FACTORY");
-        vm.label(0xa8478781D9605E17c3b4c9C79e7127c4597116E4, "Constants.FXB_FACTORY");
+        vm.label(0x7a07D606c87b7251c2953A30Fa445d8c5F856C7A, "Constants.FXB_FACTORY");
```

Comment:
cc @pegahcarter 

Curious which is current

Reply:
The change you made here to `0x7a07` is proper.

Confirmation [link](https://etherscan.io/address/0x452420df4AC1e3db5429b5FD629f3047482C543C#readContract#F11).